### DISCUSSION TITLE
Nourrir un pet dressé pour plus de récompenses

### DISCUSSION THREAD

**Discussion by Chevalier2** [Crownicles/Crownicles#690] [Nouvelles idées]
Je propose que si l'on nourrit un pet dressé, on peut obtenir plus de récompenses.

**Comment by BastLast** [comment]
Bah c'est déjà le cas -_-

**Reply by Pietagorh** [reply]
Nan mais il veut pas dire "si le pet est dressé", mais "si on nourrit un pet dressé"
Genre si tu l'as nourrit pendant la journée t'as un petit boost sur les récompenses

**Reply by BastLast** [reply]
Ah 🤔 je vois

**Comment by Chevalier2** [comment]
Je parle de **nourrir un pet dressé** pour qu'il donne plus de récompenses. Je sais qu'un pet dressé en donne plus.
